src/ws-dropdown/dropdown-menu.js

```diff
@@ -345,7 +338,10 @@ export class DropdownMenu extends Component {
         this.showChild(data);
         break;
       case 'change':
-        this.clearSelections();
+        // Clear filter input after selecting item
+        if (this.props.filterable) {
```

Comment:
not necessary, but you changed everything to this.state.filterable, I guess also here?.

Reply:
There is a tiny difference between filterable and filtered which is mentioned in the props description but maybe not clear here.
filterable -> Means that a dropdown has a filter input
filtered -> Means that the dropdown items are filtered but there is no filter input in the dropdown

The reason for this is that the ws-multi-select extends the dropdown but lays out the values and filter input. So filterable is true when <ws-dropdown filterable/> and should mean that the dropdown shows and cares about the filter value. Filtered is true when <ws-multi-select> and should mean that the filter logic of the dropdown is used but the multi select shows and cares about the filter value.

To make this more clear I'll add a more sophisticated comment here.